src/transformers/models/auto/modeling_auto.py

```diff
@@ -28,6 +28,8 @@
 MODEL_MAPPING_NAMES = OrderedDict(
     [
         # Base model mapping
+        ("decision_transformer", "DecisionTransformerModel"),
+        ("decision_transformer_gpt2", "DecisionTransformerGPT2Model"),
```

Comment:
```suggestion
```
This model type does not exist, so this shouldn't be here. It's okay if `DecisionTransformerGPT2Model` is not in an auto mapping.